salt/returners/cassandra_return.py

```diff
@@ -42,6 +48,12 @@
 # Define the module's virtual name
 __virtualname__ = "cassandra"
 
+warn_until_date(
+    "20240101",
+    "The cassandra returner is broken and deprecated, and will be removed"
+    " after {date}. Use the cassandra_cql returner instead",
+)
+
```

Comment:
Same as above, unless for returners is way less noisy.

Reply:
I can put it in `__virtual__` for the success state? Or does that also get called every time the module is loaded?

Reply:
Virtual is also called everytime the module is loaded.
But, in this case I'm ok with the noise, because the module will only load if the dependency is available right?